src/components/DatePickerInput/DatePickerInput.st.css

```diff
@@ -170,268 +124,28 @@
-/*        CalendarHeaderCaptionFont value(CurrentMonthAndYearCaptionFont),*/
-/*        CalendarHeaderCaptionFontSize 16px,*/
-/*        CalendarHeaderCaptionColor value(CurrentMonthAndYearCaptionColor),*/
-/*    */
-/*        !* month and years dropdowns *!*/
-/*        DropdownElementHeight 24px,*/
-/*        DropdownElementFont value(CurrentWeekdayFont),*/
-/*        DropdownElementFontSize 14px,*/
-/*        DropdownElementTextColor value(CurrentWeekdayTextColor),*/
-/*    */
-/*        DropdownListBGColor value(CurrentBackgroundColor),*/
-/*        DropdownListPadding 12px 0,*/
-/*    */
-/*        DropdownListOptionItemFont font("{theme: 'Body-M', size: '14px', lineHeight: '20px'}"),*/
-/*        DropdownListOptionItemColor value(CurrentDayColor),*/
-/*        DropdownListOptionItemPadding 0 20px,*/
-/*        DropdownListOptionItemBGColorHover value(DropdownOptionSelectedAndHoverBGColor),*/
-/*        DropdownListOptionItemSelectedBGColor value(DropdownOptionSelectedAndHoverBGColor),*/
-/*        DropdownListOptionItemSelectedBGColorHover value(DropdownOptionSelectedAndHoverBGColor), */
-/*        !**********************!*/
-/*    */
-/*        !* calendar Weekdays *!*/
-/*        CalendarWeekdaysPadding 0 value(sidePadding),*/
-/*        CalendarWeekdaysBGColor value(CurrentBackgroundColor),*/
-/*    */
-/*        CalendarWeekdayPadding 20px 0 16px,*/
-/*        CalendarWeekdayFont value(CurrentWeekdayFont),*/
-/*        CalendarWeekdayFontSize 14px,*/
-/*        CalendarWeekdayTextColor value(CurrentWeekdayTextColor),*/
-/*        !**********************!*/
-/*    */
-/*        !* calendar body *!*/
-/*        CalendarBodyPadding 0 value(sidePadding) 20px,*/
-/*        CalendarBodyBGColor value(CurrentBackgroundColor),*/
-/*    */
-/*        CalendarDaySize 34px,*/
-/*        CalendarDayFont value(CurrentDayFont),*/
-/*        CalendarDayFontSize 14px,*/
-/*        CalendarDayTextColor  value(CurrentDayColor),*/
-/*        CalendarDayTextColorHover color(value(DeafultDayColor)),*/
-/*        CalendarDayBGColorHover value(CurrentDayBGColorHover),*/
-/*    */
-/*        CalendarDayTodayBGColor none,*/
-/*        CalendarDayTodayTextColor value(CurrentSelectedDayBGColor),*/
-/*        CalendarDayTodayBoxShadow none,*/
-/*        CalendarDayTodayBGColorHover value(CurrentDayBGColorHover),*/
-/*        CalendarDayTodayTextColorHover value(CurrentSelectedDayBGColor),*/
-/*        CalendarDayTodayBoxShadowFocus none,*/
-/*    */
-/*        CalendarDaySelectedBGColor value(CurrentSelectedDayBGColor),*/
-/*        CalendarDaySelectedTextColor  value(CurrentSelectedDayColor),*/
-/*        CalendarDaySelectedBGColorHover applyOpacity(value(CurrentSelectedDayBGColor), 0.7),*/
-/*    */
-/*        CalendarDayDisabledColor value(CurrentDisabledDayColor),*/
-/*        CalendarDayOutsideColor value(CurrentDisabledDayColor),*/
-/*    */
-/*        !**********************!*/
-/*    );*/
-/*}*/
+.overrideStyleParams .wsrDatePicker {
```

Comment:
I still think it would be nice to try passing the calendar className to the datepicker component inside the datepickerinput, and then see if the `overrideStyleParams` will work

Reply:
Yes, I agree that we should find a solution. But let's keep it for later :) I can even create a task so we will not forget it.